### DISCUSSION TITLE
How to use it?

### DISCUSSION THREAD

**Discussion by svaldenegro** [amerkoleci/JoltPhysicsSharp#35] [Q&A]
Maybe a little piece o code to get a good startup would be nice.
Also, i'm trying to look for the built in character controller classes and can't see them.
https://jrouwe.github.io/JoltPhysics/class_character.html

**Comment by amerkoleci** [comment]
Hi,
Lot of things would be nice, specially on spare time project.

Feel free to submit PR with example or became [Sponsor](https://github.com/sponsors/accounts) so I can add those "extra" requests.

Cheers

**Reply by MrScautHD** [reply]
how much you want for implimenting it?
